.gitmodules

```diff
@@ -1,11 +1,7 @@
 [submodule "FV3"]
   path = FV3
-  url = https://github.com/NOAA-EMC/fv3atm
-  branch = develop
-[submodule "NEMS"]
-  path = NEMS
-  url = https://github.com/NOAA-EMC/NEMS
-  branch = develop
```

Comment:
Is this the code at the latest develop branch? It seems to me the code changes  in this PR are not pointing to the top of develop branch. Please check

Reply:
Sorry, I messed up the merge of .gitmodules. I will correct this and double-check that everything is still up-to-date.

Reply:
Corrected. The components were up-to-date, but the .gitmodules was not. Of course, components are constantly changing, so I'll have to keep updating them as the review process proceeds.